raddb/mods-available/yubikey

```diff
@@ -235,7 +235,7 @@ yubikey {
 			#
 			#  spread:: Enable LRU (Least Recently Used).
 			#
-			#  The `yubico` library uses `curl` multihandles. If the multihandle is reused
+			#  The `yubico` library uses `curl` multi-handles. If the multi-handle is reused
```

Comment:
It's actually 'multi handle'

Reply:
https://curl.haxx.se/libcurl/c/libcurl-multi.html

Reply:
agreed, I should have checked how CURL defines it rather than the rest of the world ;-)